nncf/torch/quantization/algo.py

```diff
@@ -1062,6 +1062,7 @@ def is_weights(ip: PTTargetPoint) -> bool:
                                             scale_shape=tuple(scale_shape),
                                             logarithm_scale=use_logarithm_scale,
                                             half_range=half_range,
+                                            export_quantized=is_weights(primary_ip),
```

Comment:
maybe `is_quantized_on_export` ? sounds more clear, but still requires some documentation. because even with this PR it was not obvious why this argument is needed.

Reply:
Name of param changed to `is_quantized_on_export` and docs were added to `PTQuantizerSpec`